### ISSUE TITLE
mt.BaseValues() examples and documentation don't match expected value in 0.1.2

### ISSUE THREAD

**Issue by murgui** [Aalto-Electric-Drives/motulator#85]
Hi,

I'm using v0.1.2, I tried running 'plot_vector_ctrl_pmsm_2kw.py' and it reports the error 

```
Traceback (most recent call last):
    base = mt.BaseValues(
TypeError: BaseValues.__init__() got an unexpected keyword argument 'n_p'
```
I checked the reference at https://aalto-electric-drives.github.io/motulator/autoapi/motulator/index.html#motulator.BaseValues and indeed the example used the suggested arguments.

I removed the argument and then the error was that it was missing argument 'p', so it looks like n_p has been renamed to p, so either the examples should be changed to p or the function should be changed to n_p.

Regards

**Comment by lauritapio** [comment]
Thanks for pointing this out! The polepair number was indeed recently named to n_p, but it seems that there are some places still where this has not been changed. We'll get this fixed shortly!

**Comment by mhinkkan** [comment]
Thank you for the valuable feedback!

The reason for the problem might be that we have updated the repository since last release quite a lot, including the change in the variable name (p -> n_p). To fix the problem, new release 0.1.3 was published a moment ago. 

Furthermore, the documentation regarding installation was updated:

https://aalto-electric-drives.github.io/motulator/installation.html

You might be interested in Option 2, i.e. cloning the repository. It allows you to modify the existing system models and the example controllers. The example instructions are for VS Code, but quite similar steps could be used with other IDEs.

Please inform us if the problem still appears after installing the latest release.
